src/utils/ExpressConfigFunctionFactory.ts

```diff
@@ -30,6 +31,8 @@ export const createExpressConfigFunction = (directory: string) => (app: express.
         express: app,
     });
 
+    nunjucksEnv.addFilter('govukDate', dateFilter);
```

Comment:
Not a big fan of this approach. What gov UK date really is? Is there such thing? I'd just call it `date`. I am curious your reason behind that choice.

Reply:
It was a temporary change to debug an issue with the pipeline, which I then forgot to revert. I'll put it back to `date`.